gradoop-flink/src/main/java/org/gradoop/flink/io/impl/dot/DOTDataSink.java

```diff
@@ -52,6 +59,19 @@
   public DOTDataSink(String path, boolean graphInformation) {
     this.path = path;
     this.graphInformation = graphInformation;
+    this.format = DotFormat.HTML;
```

Comment:
Since you are adding a new constructor below, call that one instead of setting those fields again. (`this(path, graphInformation, DotFormat.HTML);`)